src/components/Hypothesis/HypothesisTitle/index.tsx

```diff
@@ -43,22 +47,27 @@ export const HypothesisTitle = ({ node, api }: Props) => {
   const involved = Array.from(commentsMap.values());
 
   return (
-    <div className="mt-2 mr-2 mb-2 pr-8 flex flex-nowrap space-x-4 items-center">
+    <div className="mt-2 mr-2 mb-2 pr-8 flex flex-nowrap items-center">
+      <StatusDropdownContainer
+        nodeId={node?.id}
+        status={node?.status}
+        updateMutation={api?.updateMutation}
+      />
       <EditableText
         value={getValues("title")}
         sharedClassName="text-2xl font-semibold text-gray-900 grow"
         onChange={(value: string) => {
           setValue("title", value);
         }}
       />
-      <div className="flex">
-        <Avatar size="sm" user={node?.created_by} />
-        <p className="font-inter text-dark-gray font-normal text-sm ml-1.5 mt-0.5">
-          {node?.created_by?.name}
-        </p>
-      </div>
-      {Boolean(involved.length) && (
-        <AvatarGroup maxCount={5} users={involved} size="sm" />
+      {Boolean(involved.length || node?.created_by) && (
+        <span className="pl-4">
+          <AvatarGroup
+            maxCount={5}
+            users={[node?.created_by].concat(involved)}
```

Comment:
What happens when **node?.created_by** is undefined, do we filter out all the undefined entries ?